core/rawdb/accessors_chain.go

```diff
@@ -1175,6 +1199,11 @@ func ReadEtxSet(db ethdb.Reader, hash common.Hash, number uint64) types.EtxSet {
 func WriteEtxSet(db ethdb.KeyValueWriter, hash common.Hash, number uint64, etxSet types.EtxSet) {
 	var entries []EtxSetEntry
 	for etxHash, entry := range etxSet {
+		etxRlp, err := rlp.EncodeToBytes(entry)
+		if err != nil {
+			return
```

Comment:
Same here, we can either make the function return an error or just log it.

Reply:
done